ui/hidden_seek_button.js

```diff
@@ -93,8 +104,8 @@ shaka.ui.HiddenSeekButton = class extends shaka.ui.Element {
     if (!this.triggeredTouchValid_) {
       this.triggeredTouchValid_ = true;
       this.lastTouchEventTimeSet_ = Date.now();
-      this.hideSeekButtonContainerTimer_.tickAfter(1);
-    } else if (this.lastTouchEventTimeSet_+1000 > Date.now()) {
+      this.hideSeekButtonContainerTimer_.tickAfter(0.5);
+    } else if ((this.lastTouchEventTimeSet_ + 1000) > Date.now()) {
```

Comment:
I'm not sure what the purpose of `lastTouchEventTimeSet_` is, given that when `hideSeekButtonContainerTimer_` goes off it sets `triggeredTouchValid_` to false. Is there any situation where the 0.5s timer has not yet gone off, but this timestamp is over 1s old?